src/views/Text2Image/Text2ImageCustomPickerElement.vue

```diff
@@ -0,0 +1,427 @@
+<template>
+	<div class="text2image-picker-content-wrapper">
+		<div class="text2image-picker-content">
+			<h2>
+				{{ t('assistant', 'AI Image Generation') }}
+			</h2>
+			<div class="input-wrapper">
+				<NcTextField ref="text2image-search-input"
+					:value.sync="query"
+					:label="inputPlaceholder"
```

Comment:
```suggestion
					:label="inputPlaceholder"
					placeholder=""
```
In NcInputField/NcTextField, the label is used as placeholder if no placeholder is set. Let's use either an empty one or a query example.